_inc/client/setup-wizard/feature-toggle/map-feature-to-props.js

```diff
@@ -0,0 +1,1028 @@
+			}
+
+			let info;
+			if ( inCurrentPlan ) {
+				info = __( `Included with ${ sitePlan.product_name }` );
+			}
+
+			return {
+				title: __( 'Daily or Real-time backups' ),
+				details: __( 'Get time travel for your site with Jetpack Backup.' ),
+				checked: backupsActive,
+				isDisabled: inCurrentPlan,
+				optionsLink,
+				upgradeLink,
+				info,
+				isPaid: true,
+			};
+		},
+	},
+
+	'beautiful-math': {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Beautiful math' ),
+				details: __( 'Display math and formulas beautifully.' ),
+				checked: getSetting( state, 'latex' ),
+				optionsLink: '#/settings?term=latex',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { latex: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	'brute-force-protect': {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Brute force protection' ),
+				details: __( 'Stop malicious login attempts.' ),
+				checked: getSetting( state, 'protect' ),
+				optionsLink: '#/settings?term=brute%20force',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { protect: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	carousel: {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Carousel' ),
+				details: __(
+					'Create full-screen carousel slideshows for the images in your posts and pages.'
+				),
+				checked: getSetting( state, 'carousel' ),
+				optionsLink: '#/settings?term=carousel',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { carousel: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	'comment-likes': {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Comment Likes' ),
+				details: __( 'Increase engagement with liking on comments.' ),
+				checked: getSetting( state, 'comment-likes' ),
+				optionsLink: '#/settings?term=comment%20likes',
+			};
+		},
+
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { 'comment-likes': ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	comments: {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Comments' ),
+				details: __( 'An enhanced comments section with better verfiication.' ),
+				checked: getSetting( state, 'comments' ),
+				optionsLink: '#/settings?term=comments',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { comments: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	'contact-form': {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Contact Form' ),
+				details: __( 'Gutenberg ready forms!' ),
```

Comment:
"Gutenberg" feels a bit jargon to me; I don't know that most people know what that means.

Reply:
@jessefriedman any thoughts on this? I wondered the same thing when I was testing this one, I'm not sure how many users will both recognize Gutenberg and contextualize it here related to contact forms.

Reply:
Tracked here: https://github.com/Automattic/jetpack/issues/15947